packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+) => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
+
+  const alice = await devEnv.createRandomPerson();
+  // set access control conditions for encrypting and decrypting
+  const accs = AccessControlConditions.getEmvBasicAccessControlConditions({
+    userAddress:
+      generator.name === 'getEoaSessionSigs'
+        ? alice.authSig?.address!
+        : alice.authMethodOwnedPkp?.ethAddress!,
+  });
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const encryptRes = await LitJsSdk.encryptString(
+    {
+      accessControlConditions: accs,
+      dataToEncrypt: 'Hello world',
+    },
+    devEnv.litNodeClient as unknown as ILitNodeClient
+  );
+
+  // -- Expected output:
+  // {
+  //   ciphertext: "pSP1Rq4xdyLBzSghZ3DtTtHp2UL7/z45U2JDOQho/WXjd2ntr4IS8BJfqJ7TC2U4CmktrvbVT3edoXJgFqsE7vy9uNrBUyUSTuUdHLfDVMIgh4a7fqMxsdQdkWZjHign3JOaVBihtOjAF5VthVena28D",
+  //   dataToEncryptHash: "64ec88ca00b268e5ba1a35678a1b5316d212f4f366b2477232534a8aeca37f3c",
+  // }
+
+  // -- assertions
+  if (!encryptRes.ciphertext) {
+    throw new Error(`Expected "ciphertext" in encryptRes`);
+  }
+
+  if (!encryptRes.dataToEncryptHash) {
+    throw new Error(`Expected "dataToEncryptHash" to in encryptRes`);
+  }
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+            const resp = await Lit.Actions.decryptAndCombine({
+              accessControlConditions,
+              ciphertext,
+              dataToEncryptHash,
+              authSig: null,
+              chain: 'ethereum',
+            });
+            Lit.Actions.setResponse({
+                response: resp
+            });
+          })();`,
+    jsParams: {
+      accessControlConditions: accs,
+      dataToEncryptHash: encryptRes.dataToEncryptHash,
+      ciphertext: encryptRes.ciphertext,
+    },
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  expect(res?.response).toEqual('Hello world');
+};
+
+const signAndCombine = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+) => {
+  const alice = await devEnv.createRandomPerson();
+
+  const litActionSessionSigs = await generator(devEnv, alice);
+
+  const res = await devEnv.litNodeClient?.executeJs({
+    sessionSigs: litActionSessionSigs,
+    code: `(async () => {
+            const sigShare = await LitActions.signAndCombineEcdsa({
+              toSign: dataToSign,
+              publicKey,
+              sigName: "sig",
+            });
+            Lit.Actions.setResponse({
+                response: sigShare
+            });
+          })();`,
+    jsParams: {
+      dataToSign: alice.loveLetter,
+      publicKey: alice.pkp?.publicKey,
+    },
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
```

Comment:
Need this to be in a finally{} block so that if code above throws, we release the key